compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIELowerToUKernels.cpp

```diff
@@ -266,9 +265,7 @@ static bool isMatmul(linalg::LinalgOp linalgOp, AIEPassPipeline passPipeline) {
     return false;
   }
 
-  if (passPipeline == AIEPassPipeline::PadPipeline) {
-    return match2DLinalgGenericMatmul(linalgOp);
```

Comment:
This function in the file will also get eliminated - you may remove it.

Reply:
I think this function is useful. We can leave it here in case you'll need this in the future?

Reply:
In fact, we won't need both `match2D*` (Pad) and `match6D*` (Simple-Pack) - you may remove both of them - no issues.

They'll not find use anywhere else in the pipeline.

Reply:
Done.